tls/extensions/s2n_npn.c

```diff
@@ -79,3 +79,62 @@ const s2n_extension_type s2n_server_npn_extension = {
     .should_send = s2n_server_npn_should_send,
     .if_missing = s2n_extension_noop_if_missing,
 };
+
+bool s2n_npn_encrypted_should_send(struct s2n_connection *conn)
+{
+    return s2n_server_alpn_should_send(conn);
```

Comment:
is this wrapped in a new function just for clarity/naming?

Reply:
Yeah it's mostly for clarity.